core/integration/features/spot/orders/0014-ORDT-163.feature

```diff
@@ -0,0 +1,108 @@
+
+    Given the following network parameters are set:
+      | name                                    | value |
+      | network.markPriceUpdateMaximumFrequency | 1s    |
+      | market.value.windowLength               | 1h    |
+      | spam.protection.max.stopOrdersPerMarket | 5     |
+
+    Given the following assets are registered:
+      | id  | decimal places |
+      | ETH | 2              |
+      | BTC | 2              |
+
+    Given the fees configuration named "fees-config-1":
+      | maker fee | infrastructure fee |
+      | 0         | 0                  |
+    Given the log normal risk model named "lognormal-risk-model-1":
+      | risk aversion | tau  | mu | r   | sigma |
+      | 0.001         | 0.01 | 0  | 0.0 | 1.2   |
+    And the price monitoring named "price-monitoring-1":
+      | horizon | probability | auction extension |
+      | 360000  | 0.999       | 1                 |
+
+    And the spot markets:
+      | id      | name    | base asset | quote asset | risk model             | auction duration | fees          | price monitoring   | decimal places | position decimal places | sla params    |
+      | BTC/ETH | BTC/ETH | BTC        | ETH         | lognormal-risk-model-1 | 1                | fees-config-1 | price-monitoring-1 | 2              | 2                       | default-basic |
+
+    # setup accounts
+    Given the parties deposit on asset's general account the following amount:
+      | party  | asset | amount |
+      | party1 | ETH   | 100    |
+      | party1 | BTC   | 11     |
+      | party2 | ETH   | 10000  |
+      | party2 | BTC   | 10     |
+      | party3 | ETH   | 10000  |
+      | party3 | BTC   | 1000   |
+      | party4 | BTC   | 1000   |
+      | party5 | BTC   | 1000   |
+    And the average block duration is "1"
+
+    # Place some orders to get out of auction
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     |
+      | party3 | BTC/ETH   | buy  | 1      | 1000  | 0                | TYPE_LIMIT | TIF_GFA |
+      | party4 | BTC/ETH   | sell | 1      | 1000  | 0                | TYPE_LIMIT | TIF_GTC |
+
+    And the opening auction period ends for market "BTC/ETH"
+    When the network moves ahead "1" blocks
+    Then the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "BTC/ETH"
+    And the mark price should be "1000" for the market "BTC/ETH"
+
+  Scenario:0014-ORDT-163, 0014-ORDT-164: A wrapped buy/sell order will be rejected when triggered if the party doesn't have enough of the required quote asset to cover the order.
+    Given the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     |
+      | party1 | BTC/ETH   | sell | 10     | 1000  | 0                | TYPE_LIMIT | TIF_GTC |
+      | party2 | BTC/ETH   | buy  | 10     | 1000  | 1                | TYPE_LIMIT | TIF_GTC |
```

Comment:
Could we maybe simplify the test a little by removing this step? There is no requirement for the parties to have made any trades to place a stop order on a spot market since there is no concept of a position.

Reply:
If we do that then the balances at the end should match the deposited balances which I think is easier to reason - with this trade we should at least check the balances after the trade.